### PR TITLE
Add note about migrations run by background jobs

### PR DESCRIPTION
**Pull request by SystemKeeper**
### 🖼️ Screenshots

image

### CODE REVIEW

File: admin_manual/maintenance/upgrade.rst [hunk 1]
```diff
@@ -28,6 +28,14 @@ Nextcloud must be upgraded step by step:
   * **You cannot skip major releases.** Please re-run the upgrade until you have reached the highest available (or applicable) release.
   * Example: 18.0.5 -> 18.0.11 -> 19.0.5 -> 20.0.2
 
+**Wait for background migrations to finish after major upgrades**. After upgrading to a new major version, some migrations are scheduled to run 
+as a background job. If you plan to upgrade directly to another major version (e.g. 24 -> 25 -> 26) you need to make sure these
+migrations were executed before starting the next upgrade. To do so you should run the ``cron.php`` file 2-3 times, for example::
+
+ $ sudo -u www-data php -f /var/www/nextcloud/cron.php
+
+For more information about background jobs see :doc:`../configuration_server/background_jobs_configuration`.
```
Comment (ChristophWurst) [inline admin_manual/maintenance/upgrade.rst:37]: nit: use label and `:ref:` for stable references even if pages move

**Review by ChristophWurst — APPROVED** [review]

File: admin_manual/maintenance/upgrade.rst (hunk 1 above)
Comment (SystemKeeper) [reply admin_manual/maintenance/upgrade.rst:37]: Sorry, just noticed your comment.

You mean like this?

```diff
diff --git a/admin_manual/configuration_server/background_jobs_configuration.rst b/admin_manual/configuration_server/background_jobs_configuration.rst
index 69568564b..b5c67a644 100644
--- a/admin_manual/configuration_server/background_jobs_configuration.rst
+++ b/admin_manual/configuration_server/background_jobs_configuration.rst
@@ -1,3 +1,5 @@
+.. _background-jobs-configuration:
+
 ===============
 Background jobs
 ===============
diff --git a/admin_manual/maintenance/upgrade.rst b/admin_manual/maintenance/upgrade.rst
index aa209a933..c6a3ccd45 100644
--- a/admin_manual/maintenance/upgrade.rst
+++ b/admin_manual/maintenance/upgrade.rst
@@ -34,7 +34,7 @@ migrations were executed before starting the next upgrade. To do so you should r
 
  $ sudo -u www-data php -f /var/www/nextcloud/cron.php
 
-For more information about background jobs see :doc:`../configuration_server/background_jobs_configuration`.
+See ref:`background jobs configuration<background-jobs-configuration>` for more information.
 
 **Upgrading is disruptive**. Your Nextcloud server will be put into maintenance
 mode, so your users will be locked out until the upgrade is completed. Large
```

Would do this in a follow-up then.

**Pull request merged**